scss/core/abstracts/_colors.scss

```diff
@@ -93,6 +93,9 @@ $teal: #24A4B1 !default;
 
 $dodger-blue: #A3BAE0 !default;
 
+// Color used in terminal header and footers
+$terminal-blue: #3D51AD !default;
```

Comment:
add color in `colors.js` and `colors.mdx` file as well. This is to display in the colors screens https://awesome-thompson-1f5f82.netlify.app/#/products-colors
<img width="937" alt="Screenshot 2020-11-26 at 11 54 50 AM" src="https://user-images.githubusercontent.com/34312966/100315179-38601c00-2fde-11eb-8dd8-02614fbcb310.png">